### PR TITLE
docs: update 5.13.0 release notes

### PR DESCRIPTION
**Pull request by devcorpio**
- Release notes for 5.13.0

**PREVIEW**

preview

--

Additionally, I updated the docs for Angular, preview here:

Screenshot 2023-07-19 at 16 24 46

P.S. Once this is merged, I will also update the docs for 5.x branch

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -29,6 +29,22 @@ See Conventional Commits (https://conventionalcommits.org) for commit guidelines
...
+==== 5.13.0 (2023-07-19)
+[float]
+===== Potentially breaking changes
+* Previously, breakdown timings were reported as milliseconds values in the `span.self_time.sum.us` field, which was wrong. From now on, it will be reported as microsecond values to match the APM specification: {pull}1381[#1381]
+* The metrics `transaction.duration.sum.us`, `transaction.duration.count` and `transaction.breakdown.count` are no longer recorded: {pull}1382[#1382]
+
+===== Features
+* Support for Angular 16 is made available via `@elastic/apm-rum-angular@3.0.0`: {pull}1380[#1380]
+* Improve user-interaction transactions naming. Unsupported browsers: IE11 {pull}1390[#1390]
+* Add `apmRequest` config to TypeScript typings: {pull}1254[#1254]
```
Comment (vigneshshanmugam) [inline CHANGELOG.asciidoc:42]:
```suggestion
* Add `apmRequest and sendCredentials` config to TypeScript typings: {pull}1254[#1254] {pull}1243[#1243]
```

Comment (vigneshshanmugam) [reply CHANGELOG.asciidoc:42]: Lets merge this both.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -29,6 +29,22 @@ See Conventional Commits (https://conventionalcommits.org) for commit guidelines
...
+* Add `sendCredentials` config to TypeScript typings: {pull}1243[#1243]
+
+===== Bug fixes
+* Check if a performance metric is available before trying to measure it: {pull}1246[#1246]
```
Comment (vigneshshanmugam) [inline CHANGELOG.asciidoc:46]:
```suggestion
* fix warnings where performance entries are not supported: {pull}1246[#1246]
```

**Review by vigneshshanmugam — APPROVED** [review]

**Comment by devcorpio** [comment]
thanks for the suggestions, @vigneshshanmugam. This is the preview after the changes:

Screenshot 2023-07-19 at 17 17 04

**Pull request merged**